python/pyspark/ml/feature.py

```diff
@@ -565,6 +607,20 @@ def vocabulary(self):
         """
         return self._call_java("vocabulary")
 
+    @since("2.4.0")
+    def setMinTF(self, value):
```

Comment:
If we're going to have the setters in both the model and the estimator maybe we should consider putting it in the shared params class?

Reply:
I agree but I was trying to match the Scala API.  My only thought is it was done this way to leave it up to the implementations if they allow setting the params.  What do you think?

Reply:
Sounds reasonable to me.